README.md

```diff
@@ -110,10 +136,21 @@ All four types of casts come in two flavors. One needs the targetted value type,
   unlib::scale_cast<unlib::milli>(any_weight);
 ```
 
-The other does not need this. It returns a temporary object from which a quantity can be created, and that can be assigned to a quantity. Depending on the quantity created from it, or it is assigned to, the respective conversion is invoked automatically: 
+The other flavor of the same cast does not need this. It returns a temporary object from which a quantity can be created, and that can be assigned to a quantity. Depending on the quantity created from it, or it is assigned to, the respective conversion is invoked automatically: 
+
+```c++
+  my_floating_hours float_hrs = unlib::tag_value(integer_seconds); // note: scale cast from secs to hrs is implicit
```

Comment:
```suggestion
  my_floating_hours float_hrs = unlib::value_cast(integer_seconds); // note: scale cast from secs to hrs is implicit
```

Reply:
It seems I modified the wrong part of the cast's identifier when copying this from elsewhere... `:-/`